javascript/example/webgl_loader_draco.html

```diff
@@ -41,8 +41,8 @@
         </div>
         <pre id="fileDisplayArea"><pre>
     </div>
-    <script src="three.js"></script>
-    <script src="draco_decoder.js"></script>
+    <script src="https://cdn.rawgit.com/mrdoob/three.js/r84/build/three.min.js"></script>
+    <script src="../draco_decoder.js"></script>
```

Comment:
this relative path is inaccessible to the server when it is running from the javascript/example/ folder (at least to the http server that we suggest in our documentation: `python -m SimpleHTTPServer`